Core/AppUrls.swift

```diff
@@ -22,26 +22,29 @@ import Foundation
 public struct AppUrls {
 
     private struct Url {
-
-        // You can change this to use a subdomain for testing (e.g. "test.")
-        static let subdomain = ""
+        
+        static var devMode: String {
+            return isDebugBuild ? "?test=1" : ""
+        }
+        
         static let domain = "duckduckgo.com"
-        static let base = "\(subdomain)\(domain)"
-        static let home = "https://\(base)"
-        static let favicon = "\(home)/favicon.ico"
-        static let autocomplete = "\(home)/ac/"
-        static let disconnectMeBlockList = "\(home)/contentblocking.js?l=disconnect"
-        static let easylistBlockList = "\(home)/contentblocking.js?l=easylist"
-        static let easylistPrivacyBlockList = "\(home)/contentblocking.js?l=easyprivacy"
-        static let httpsUpgradeList = "\(home)/contentblocking.js?l=https2"
-        static let trackersWhitelist = "\(home)/contentblocking/trackers-whitelist.txt"
-        static let surrogates = "\(home)/contentblocking.js?l=surrogates"
-        static let atb = "\(home)/atb.js"
-        static let exti = "\(home)/exti/"
-        static let feedback = "\(home)/feedback.js?type=app-feedback"
-        static let faviconService = "\(home)/ip3/%@.ico"
         
-        static let pixel = "https://improving.\(domain)/t/%@_ios_%@"
+        static let base = ProcessInfo.processInfo.environment["BASE_URL", default: "https://\(domain)"]
```

Comment:
Got rid of `home` because I needed to be able to change the scheme as well as the domain, otherwise there would be several of these replacements and this is simpler.